src/content/docs/getting-started/intro.md

```diff
@@ -3,27 +3,51 @@ title: Intro
 description: An overview of Membrane and its uses
 ---
 
-<a href="https://www.membrane.io">Membrane</a> is a fast, powerful way to write internal tools in TypeScript, connecting the apps you already use at work.
+<a href="https://www.membrane.io">Membrane</a> is a new and powerful way to quickly build applications, utilities
+and internal tools with TypeScript, while integrating with the apps and services you already use.
 
-Popular use cases for Membrane include automations using APIs, email, text messages, long-lived workflows, cron jobs, webhook handlers, web scraping, chat bots, and many more.
+To effectively use Membrane, we recommend playing around with the [Hello World](/getting-started/hello-world) program and then learning
+the [core concepts](/concepts/programs) that make Membrane unique and powerful.
 
 <!-- TODO: Add a visualization that communicates what membrane is in a few seconds -->
 
 ## What's different?
 
 Membrane has some superpowers that differentiate it from other JavaScript/TypeScript environments, most notably:
 
-- _Programs are durable_: there's no need to store & load data from a database—just keep data you want to save in the `state` object.
-- _Access APIs via a unified graph_: use external APIs in a standard way without worrying about API quirks and reading lots of docs.
-- _Everything is in the logs_: Our runtime uses a write-ahead log, so if it's not in the logs, it didn't happen.
-- _Programmable email_: every program comes with an email address. Simply export an `email` function to programmatically handle incoming email.
+- **Programs are durable** There's no need to store & load data from a database. Instead, keep everything in the `state` object.
+- **Access everything via a unified graph** Use external APIs in a standard way without worrying about API quirks and reading lots of docs.
+- **Everything is in the logs** Our runtime uses a write-ahead log, so if it's not in the logs, it didn't happen.
+- **Built-in email** Every program comes with an email address. Simply export an `email` function to programmatically handle incoming email.
 
-Check out our <a href="https://www.membrane.io">homepage</a> for a full showcase of features & examples.
+Start learning more in the [Durable Programs](/concepts/programs) section.
+
+## When to use Membrane
+
+Popular use cases for Membrane include
+
+- Internal tools
+- Long lived workflows
+- Email or SMS automation
+- Slack/Discord/Telegram bots
+- LLM-powered AI agents
+- Webhook handlers
+- Cron jobs
+- Simple HTTP APIs
+- Simple server-side rendered websites
+- Personal utilities
+- Prototypes
+
+## When not to use Membrane
```

Comment:
Could you maybe explain why this section "When not to use Membrane" is necessary? Is it common to include a "When not to use" section?

Reply:
I like seeing when-not-to-use sections, like [Buttondown's](https://buttondown.com/alternatives). It's realistic and shows that the developers (us) are realistic because no tool is perfect for everyone.

Reply:
I'm convinced. Sounds good!